src/custom-element.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import * as dom from './dom';
+import * as newLoader from '../src/loader-new';
```

Comment:
Can we avoid (new) wildcard imports? I believe it's not native ES6 and may have bundling implications.

Reply:
It is native ES6, but does complicate DCE for naive tools. Closure handles it fine, but out custom ESLint rules, etc, have more difficulty.

Note that you **must not** mutate the wildcard object. That's forbidden in ES6, but allowed in transpilation.

Reply:
I really like the namespacing aspect here. I find importing random functions with names that, outside of their module context, make no sense bad for readability. Also I don't want to prefix my module's function names with their module name.

I wish we could do something like:
```
import {foo, bar} as myModule from 'module.js'
```
(@jridgewell is something like this being considered in TC39?)

Anyway, are we okay with this or should I get rid of it?

Reply:
(all that being said, since I merged legacy and new loader files, I renamed functions and importing individual exports now)